docs/components/cards.md

```diff
@@ -13,12 +13,11 @@ eleventyNavigation:
 ## Best Practices
 Cards are designed to be flexible enough to accommodate most content types
 - If a Card occurs outside of a [Content Container](/components/content_container/), it will automatically get a white background 
-- If the card occurs inside of a [Content Container](/components/content_container/) it will automatically get a gray background
+- If the card occurs inside of a Content Container it will automatically get a gray background
```

Comment:
Is there a good reason not to link in both places?

Reply:
@james-alt bump